community_supplied/Interface/snmp-interface-throughput.rule

```diff
@@ -1,11 +1,282 @@
+            */
+            field out-bw-util {
+                formula {
+                    eval {
+                        expression "($out-mbps / $if-speed ) * 100";
+                    }
+                }
+                type float;
+                description "Output bandwidth utilization";
+            }
+            field out-mbps {
+                formula {
+                    rate-of-change {
+                        field-name "$output-bytes";
+                        multiplication-factor 0.000001;
+                    }
+                }
+                type float;
+            }
+            field predicted-in-bw-util {
+                formula {
+                    predict {
+                        algorithm median-prediction;
+                        learning-period 1d;
+                        pattern-periodicity 1h;
+                        field-name "$in-bw-util";
+                        prediction-offset 1d;
+                    }
+                }
+                type float;
+                description "ML predicted input bandwidth utilization";
+            }
+            field predicted-out-bw-util {
+                formula {
+                    predict {
+                        algorithm median-prediction;
+                        learning-period 1d;
+                        pattern-periodicity 1h;
+                        field-name "$out-bw-util";
+                        prediction-offset 1d;
+                    }
+                }
+                type float;
+                description "ML predicted output bandwidth utilization";
+            }
+            /*
+            * Fields defined with variables and constant values
+            */
+            field threshold {
+                constant {
+                    value "{{threshold}}";
+                }
+                type float;
+                description "Interface bandwidth utilization increase threshold";
+            }
+            /*
+            * Anomaly detection logic.
+            */
+            trigger in-bw-utilization {
+                synopsis "Interface input traffic kpi";
+                description "Sets health based on input traffic exceed threshold";
+                frequency 180s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * the input throughput utilization ($in-bw-util %) is greater than
+                * the threshold ($threshold %).
+                */
+                term is-input-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$in-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name input throughput utilization ($in-bw-util %) crossed threshold value ($threshold %). Mbps:$in-mbps Bps:$input-bytes bps";
```

Comment:
it (Bps:$input-bytes bps) is bytes not bps hence same. cane be removed

Reply:
Addressed and changed the message from "Bps:$input-bytes bps" to "Input-bytes:$input-bytes"